src/IRrecv.cpp

```diff
@@ -307,6 +307,35 @@ void IRrecv::setTolerance(const uint8_t percent) {
 // Get the base tolerance percentage for matching incoming IR messages.
 uint8_t IRrecv::getTolerance(void) { return _tolerance; }
 
+#if ENABLE_HIGHPASS_FILTER_OPTION
```

Comment:
Since we're only allowing high-period, low-_frequency_ pulses I'd suggest to change to
```suggestion
#if ENABLE_LOWPASS_FILTER_OPTION
```
or
```suggestion
#if ENABLE_DEGLITCHER_FILTER_OPTION
```

Reply:
Yes, you are correct. I used a poor name. I've changed it to a more generic `ENABLE_NOISE_FILTER_OPTION` and altered the procedure name as well.